meta/main.yml

```diff
@@ -9,9 +9,9 @@ galaxy_info:
   platforms:
     - name: EL
       versions:
-        - 7
+        - 9
   namespace: ome
   galaxy_tags: []
 
 dependencies:
-  - role: ome.basedeps
```

Comment:
Why is the removal of the prefix necessary?

Reply:
> Why is the removal of the prefix necessary?

Tried to return the prefix, but the test fails https://github.com/pwalczysko/ansible-role-cli-utils/actions/runs/6563940702/job/17829278386

```
galaxy_info:
role_name: my_name  # if absent directory name hosting role is used instead
  File "/opt/hostedtoolcache/Python/3.8.18/x64/bin/molecule", line 8, in <module>
namespace: my_galaxy_namespace  # if absent, author is used instead

Namespace: https://galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespace-limitations
Role: https://galaxy.ansible.com/docs/contributing/creating_role.html#role-names

As an alternative, you can add 'role-name' to either skip_list or warn_list.
...
```

Reply:
The error in https://github.com/pwalczysko/ansible-role-cli-utils/actions/runs/6563940702/job/17829278386 is expected. The role metadata includes a namespace (`ome`) so prefixing `role_name` with `ome.` will create a duplicate prefix.

My question was rather about reverting the prefix removal in `dependency.role`

Reply:
Returned the ome. prefix for the role itself (i.e `ome.cli_utils`).
This is as per this mornings discussion.